lib/ts/recipe/emailverification/index.ts

```diff
@@ -25,10 +25,25 @@ export default class Wrapper {
 
     static EmailVerificationClaim = EmailVerificationClaim;
 
```

Comment:
Why did you move the fetching email part into the index file? (i don't mind it, but any specific reason why?)

Reply:
I didn't really move it - the claim and the API implementations still do the fetching themselves. I made the email optional because this way it matches the auth recipe specific methods, so it'd be an easier update for people (and I didn't have to update that many tests)

Reply:
ok makes sense.